### ISSUE TITLE
The repositories can not be sorted properly

### ISSUE THREAD

**Issue by GyeonSeon** [goharbor/harbor#13578]
The repositories can not be sorted properly

**Issue:**
When over 15 repositories in an project, the lists are can not be sorted properly.

**Versions:**
- harbor version: 2.1.1
- docker engine version: 19.03.13
- kubernetes version: 1.19.4

**Additional context:**
39 repoistories in the project are listed like below.
```
< page 1 >
mirror/jenkins/jenkins
mirror/netapp/trident
mirror/netapp/trident-autosupport
mirror/netapp/trident-operator
mirror/prom/alertmanager
mirror/prom/prometheus
mirror/prom/pushgateway
mirror/rhel6
mirror/rhel7
mirror/rhel7-init
mirror/sonar-scanner-cli
mirror/sonarqube
mirror/sonatype/nexus3
mirror/ubuntu
mirror/weaveworks/scope

< page 2 >
mirror/cilium/cilium
mirror/cilium/hubble-relay
mirror/cilium/operator-generic
mirror/curl
mirror/docker
mirror/fluent/fluentd
mirror/goharbor/chartmuseum-photon
mirror/goharbor/harbor-core
mirror/goharbor/harbor-jobservice
mirror/goharbor/harbor-portal
mirror/goharbor/harbor-registryctl
mirror/goharbor/nginx-photon
mirror/goharbor/registry-photon
mirror/grafana/grafana
mirror/jboss/keycloak

< page 3 >
mirror/alpine
mirror/bitnami/mariadb
mirror/bitnami/nginx
mirror/bitnami/nginx-ingress-controller
mirror/bitnami/postgresql
mirror/bitnami/redis
mirror/bitnami/redis-sentinel
mirror/busybox
mirror/centos
```

**Comment by ywk253100** [comment]
The repositories are sorted by creation time in the backend, @AllForNothing dose UI sort the specific page again when get the response data?

**Comment by Hokwang** [comment]
@AllForNothing please check this issue. I am looking forward to fixing this issue in v2.1.2.

**Comment by AllForNothing** [comment]
@GyeonSeon @Hokwang 
By current design, all the filters and sorters in Datagrid column are only effective for the current page.

Because backend APIs do not yet support specifying sort fields and sorting methods.

We are considering to improve this in the future.

**Comment by Hokwang** [comment]
@AllForNothing 
Re-producing way is simple, click sort, click next page.
can't believe harbor does not have API.

can you reconsider that give higher priority to this? I think this should add version 2.2.0.

**Comment by Hokwang** [comment]
@AllForNothing @ywk253100 
I upgraded version to v2.1.3, I clicked "Name" for sorting by name, but it does not work properly.
I saw your comment and you think that sorting by name works, but it's not.

**Comment by AllForNothing** [comment]
@Hokwang 
Currently, it doesn't work, we are planning to fix this issue in v2.3

**Comment by Hokwang** [comment]
@AllForNothing OK, I left a comment because it seems that you think that sorting by name works normally.
so now, sorting does not work for every fields.
